python/core/looping/using-else-in-loops.md

```diff
@@ -83,7 +83,7 @@ a
 ---
 ## Practice
 
-Fill in the following snippet such that it will return *"Computation Done"* when the while terminates:
+Fill in the following snippet such that it will print *"Computation Done"* when the while loop terminates:
```

Comment:
I think this will look better
```suggestion
Fill in the following snippet such that it will print `"Computation Done"` when the while loop terminates:
```